src/pages/signup.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import {
+  MDBContainer,
+  MDBRow,
+  MDBCol,
+  MDBCard,
+  MDBCardBody,
+  MDBInput,
+  MDBBtn,
+} from 'mdbreact';
+import './signup.scss';
+
+function SignPage() {
+  return (
+    <MDBContainer>
+      <MDBRow>
+        <MDBCol md="5" className="mt-5 ml-5">
+          <MDBCard className="mt-5 log">
+            <MDBCardBody className="mx-4">
+              <div className="text-center">
+                <h3 className="dark mb-3">
+                  <strong>Sign up</strong>
+                </h3>
+              </div>
+              <MDBInput
+                label="Full Name"
```

Comment:
user the i18n package to translate the static text the same apply for all labels